seed/views/v3/labels.py

```diff
@@ -0,0 +1,111 @@
+# !/usr/bin/env python
+# encoding: utf-8
+"""
+:copyright (c) 2014 - 2020, The Regents of the University of California, through Lawrence Berkeley National Laboratory (subject to receipt of any required approvals from the U.S. Department of Energy) and contributors. All rights reserved.  # NOQA
+:author 'Piper Merriam <pmerriam@quickleft.com>'
+"""
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework.parsers import JSONParser, FormParser
+from rest_framework.renderers import JSONRenderer
+
+from seed.decorators import DecoratorMixin
+from seed.filters import (
+    LabelFilterBackend,
+    InventoryFilterBackendWithInvType,
+)
+from seed.models import (
+    StatusLabel as Label,
+)
+from seed.serializers.labels import (
+    LabelSerializer,
+)
+from seed.utils.api import drf_api_endpoint
+from django.utils.decorators import method_decorator
+from seed.utils.api_schema import AutoSchemaHelper
+from seed.utils.viewsets import SEEDOrgNoPatchOrOrgCreateModelViewSet
+
+
+@method_decorator(
+    name='retrieve',
+    decorator=swagger_auto_schema(
+        manual_parameters=[AutoSchemaHelper.query_org_id_field(required=False)]
+    ),
+)
+@method_decorator(
+    name='list',
+    decorator=swagger_auto_schema(
+        manual_parameters=[AutoSchemaHelper.query_org_id_field(required=False)]
+    ),
+)
```

Comment:
There's a chance we still want to keep the custom logic from the previous iteration of this list endpoint. Let's chat about this one tomorrow. We should probably review what that custom logic was allowing us to do. (See comment about the `_get_labels` method in this file.)